README.md

```diff
@@ -210,6 +215,9 @@ For xiSEARCH results there some more columns that are important for the mzIdentM
 In the "FDR settings", one can perform the actual FDR filtering. 
 
 By default, the view is set to "reduced FDR", which shows just the basic settings. The cutoff is set at 5% at the residue pair level, meaning the  error will be propagated so that 5% of the residue pairs correspond to a wrong/random match. The "boosting" features is enabled (see below for more details). These are perfectly acceptable FDR filtering settings for experiments aimed at characterising the crosslinks in a purified protein complex and should give a good idea of the number of crosslinks detectable with reasonable certainty in the sample. In analyses of cellular fractions or searches with hundreds of proteins in the database, it is advisable to also include an FDR cutoff at the "Protein Pairs" level. Similarly, in analyses devoted to method development on the quality of spectra, a filter at the "PSM" is advised.
+Whenever multiple cutoffs are selected (e.g. both residue pairs and PPI level at 5%), xiSEARCH will report only the entries at lower levels that also pass the upper level. 
+For example, only the residue pairs that will pass both the residue pair and PPI level thresholds will be considered as passing the FDR.
```

Comment:
Maybe a adding bit of a warning:
```suggestion
Whenever multiple cutoffs are selected (e.g. both residue pairs and PPI level at 5%), xiSEARCH will report only the entries at lower levels that also pass the upper level. 
For example, only the residue pairs that will pass both the residue pair and PPI level thresholds will be considered as passing the FDR. Note that the in result there will be fewer decoy residue pairs then the residue pair indicates however one can not use the still present decoys as an estimates of remaining false positive positive residue pairs (at this point decoys FDR would underestimate the FPR). 
```

Reply:
Sorry the english is not clear here.

I think you should revise to

Notice that in these cases the results files at lower levels will have less decoy lines than the stated FDR threshold at that particular level (as only the targets and decoys passing also the higher level are reported). Thus, these files cannot be used to recompute an estimate of the false positive rate, as the decoy-based FDR would drastically underestimate the FPR. Thus: with both residue pair and PPI level FDR, the number of decoys in the "Links" csv cannot be used as a basis of analyses that recompute apparent decoy-based FDR. For these analyses, the use of a threshold at a single level (the one of interest) is recommended.

Reply:
edited PR now